Assets/HoloToolkit/Input/Scripts/InputManager.cs

```diff
@@ -7,9 +7,8 @@
 using UnityEngine;
 using UnityEngine.EventSystems;
 
-#if UNITY_WSA
```

Comment:
Why are we unwrapping this for now?

Reply:
Just for now. The dev branch didn't have this wrapped and we've got to go back and fix this.